pkg/proxy/policy/policy.go

```diff
@@ -129,7 +152,7 @@ func (rule URLMatchRule) Allows(req *http.Request) bool {
 		// Avoid matching partial domain names and only match full domain parts.
 		// That is, notgoogle.com must not match google.com, but is.google.com matches google.com.
 		host := rule.Host
-		if !strings.HasPrefix(".", host) {
+		if !strings.HasPrefix(host, ".") {
```

Comment:
oof let's highlight this issue with a separate PR. better to have it be obvious in the git blame and changelog. should also add a test to confirm the fixed behavior